src/redux/books/books.js

```diff
@@ -0,0 +1,38 @@
+const ADD = 'BOOKSTORE/books/ADD';
+const DELETE = 'BOOKSTORE/books/DELETE';
+// INITIAL STATE
+
+export const initialState = [];
+
+// BOOK REDUCER
+
+const bookReducer = (state = initialState, action) => {
```

Comment:
- [ ] As requested  by the previous reviewer kindly use the ducks pattern for the export fuctions e.g
![image](https://user-images.githubusercontent.com/81932478/193713175-b5a0a8cf-470a-481d-b3e9-d5ca7b67528f.png)
- Kindly do this for the functions in the `categories,js` as well